### ISSUE TITLE
feat (YT): Move "SponsorBlock & Return YouTube Dislike Settings" inside RVX Settings. 

### ISSUE THREAD

**Issue by Aryan-Chauhan** [anddea/revanced-patches#251]
### Application

YouTube

### Feature description

Please move "SponsorBlock Settings" & "RYD Settings" inside RVX Settings.

Thanks!

### Motivation

1. It's a logical move, because they always should've been inside RVX Settings.

2. YT ReVanced also have both of them inside ReVanced Settings.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by Aryan-Chauhan** [comment]
Closing this issue, because I listed all my YT feature requests in this single issue:

#225

**Comment by anddea** [comment]
Don't you think it'll be cluttered in RVX settings? We decluttered main settings because it was messy.

**Comment by Aryan-Chauhan** [comment]
> Don't you think it'll be cluttered in RVX settings? We decluttered main settings because it was messy.

I think, RVX Settings is where they belong & will stay quite happy there, too, next to folders like General, Ads, Player, Overlay Buttons, etc.

Both of them are RVX-specific features & doesn't exist in stock app, so it's rational to put them inside. People expect them to be there, so it won't be cluttered.

They also feel out of place in a list that has stock app settings like General, Account, Your data in YouTube, etc.

That's why they should be put inside.

**Comment by Francesco146** [comment]
> Don't you think it'll be cluttered in RVX settings? We decluttered main settings because it was messy.

i agree with @Aryan-Chauhan, it won't be a clutter, but a reorganization of existing settings. i suggest also fragment the settings a little more, to better group them into separate categories. adding a search bar to the RVX main category might be the next step, as we add more and more settings.

**Comment by Aryan-Chauhan** [comment]
> > Don't you think it'll be cluttered in RVX settings? We decluttered main settings because it was messy.
> 
> i agree with @Aryan-Chauhan, it won't be a clutter, but a reorganization of existing settings. i suggest also fragment the settings a little more, to better group them into separate categories. adding a search bar to the RVX main category might be the next step, as we add more and more settings.
> 
> i take the chance to ask you another thing; in the `Tuck away preferences` patch, you commented out the "Your data in YouTube" setting, why is that? it's linked to something and can't be removed? ty! :)

Because those 3 are the only YT app stock settings I'm left with.

When anddea adds support for hiding "Account" & "Your data in YouTube" (whenever that is, no ETA required), I will hide them in my RVX.

https://github.com/anddea/revanced-patches/discussions/128#discussioncomment-8973188

![Screenshot_2024-04-03-20-30-16-95_f22faf49515899f9781ec174a4d85388.jpg](https://github.com/anddea/revanced-patches/assets/91815107/a48516f6-d34e-40b7-bca7-a4dbd326e91e)

**Comment by anddea** [comment]
> in the `Tuck away preferences` patch, you commented out the "Your data in YouTube" setting, why is that? it's linked to something and can't be removed? 

It causes a crash just like "Account". Don't know if it's related somehow (because didn't have much time to investigate yet), but both of them doesn't appear immediately if you clean install, they appear only after restarting the app. 

At the same time, I'm trying to find Smali code for YouTube settings to rewrite the patch as an option inside RVX settings but I have even less time to do that.

**Comment by anddea** [comment]
> I'll try at least! ;)

Sure, the more developers we have the better.

**Comment by Aryan-Chauhan** [comment]
This feature is now available in inotia00's RVX.
